refactored news page to align the tech stack

Lemon1903 opened the pull request:
### Tasks done:
- remove dependency to `news.module.css` and change it to tailwindcss
- refactored the HTML structure which includes removing of unnecessary `div` and merging some inner `div` with outer `div`
- commented out `newsDataMain` constant temporarily to prevent being removed automatically by my formatter
- changed the `fontFamily` property values in `tailwind.config.js` to a single string only

### Questions:
- what fonts should be used at the moment? the one defined in `typography.css`, `fonts.ts`, or the predefined fonts of tailwind?

Karmotrine commented:
Q1: Go for `font.tsx` since it uses `@next/font/local` alongside with tailwind. 
Check out the `variable` param of each font object in `font.tsx`

Karmotrine reviewed (changes requested):
Can you also include `RoundButton from "@/components/base/RoundButton";` to be refactored?
Also, please try to use `font.tsx` instead of `typography.css`
Afterwards, clear up the following:
- `src\styles\base\roundbutton.module.css`
- `src\styles\newspage\news.module.css`
- `src\styles\base\typography.css`

Lemon1903 commented:
if I'm going to use the `font.ts` file, there is no `Inter` font there. What font should I use instead?

Karmotrine commented:
> if I'm going to use the `font.ts` file, there is no `Inter` font there. What font should I use instead?

Try using karla instead. Inter is default figma font and I think it is only placeholder for low fidelity design.

Lemon1903 commented:
### Revisions made:
- migrated from`typography.css` to using next fonts in `fonts.tsx`.
- changed export of `tailwind.config.cs` from `CommonJS Module` to `ES Module` to align what is defined in `tsconfig.json`
- moved `fontFamily` property in `tailwind.config.js` outside of `extend` so that it would be the only fonts to be used in the project
- deleted some `module.css` files in `src/styles` as it is not needed anymore

Karmotrine reviewed (approved):
LGTM 🥳
